certified-connectors/My Acclaro/apiDefinition.swagger.json

```diff
@@ -0,0 +1,13061 @@
+                      "type": "string",
+                      "description": "URL"
+                    },
+                    "created": {
+                      "type": "string",
+                      "description": "Date created"
+                    },
+                    "modified": {
+                      "type": "string",
+                      "description": "Date modified"
+                    },
+                    "lastactivity": {
+                      "type": "string",
+                      "description": "Date for last activity"
+                    },
+                    "team": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "title": {
+                            "type": "string",
+                            "description": "title"
+                          },
+                          "firstname": {
+                            "type": "string",
+                            "description": "First name"
+                          },
+                          "lastname": {
+                            "type": "string",
+                            "description": "Last name"
+                          },
+                          "phone": {
+                            "type": "string",
+                            "description": "Phone number"
+                          },
+                          "email": {
+                            "type": "string",
+                            "description": "Email address"
+                          }
+                        }
+                      },
+                      "description": "team"
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "401",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "error": {
+                  "type": "string",
+                  "description": "error"
+                },
+                "error_description": {
+                  "type": "string",
+                  "description": "Error description"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/info/order-types": {
+      "get": {
+        "summary": "Get all supported order types",
+        "tags": [
+          "info"
+        ],
+        "description": "Returns all order types available for the account as an array of Order Types, each order type consisting of a name and a flag “default” which will be zero or one, indicating which order type will be set when none is specified. Note that order types is an optional argument in create order(/orders) and edit order (/orders/{orderId}) intended to assist clients who require this level of order classification for reporting purposes.",
```

Comment:
This description is too long. Please try to rephrase it. You can always check our [string guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance) for more information.

Reply:
fixed